src/targets/__tests__/maven.test.ts

```diff
@@ -109,10 +119,8 @@ describe('publish', () => {
       async () => void callOrder.push('closeAndRelease')
     );
 
-    const version = '1.0.0';
     const revision = 'r3v1s10n';
-
-    await mvnTarget.publish(version, revision);
+    await mvnTarget.publish('1.0.0', revision);
```

Comment:
Why change to directly passing in the string vs. a variable like before?

Reply:
The version isn't being used anywhere else, so IMO removes a bit of noise (like in others `upload` calls). Do you think it's easier to read by leaving the variable?

Reply:
No I have no strong opinions, just wondering. I agree it helps reduce noise in the test file so I would keep it 👍